src/stories/SpinButton/AriaSpinButton.jsx

```diff
@@ -7,50 +7,77 @@ const cx = classNames.bind(styles);
 
 const MIN = 0;
 const MAX = 100;
-/* 14. ~ 15. 키보드 Page Down, Page Up 키를 통해 감소, 증가시킬 간격 (1보다 크게 지정) */
+const STEP = 1;
+/* 15. ~ 16. 키보드 Page Down, Page Up 키를 통해 감소, 증가시킬 간격 (1보다 크게 지정) */
 const PAGE_STEP = 5;
 
 function AriaSpinButton() {
   const [value, setValue] = useState(0);
+  const [shouldReadValue, setShouldReadValue] = useState(false);
```

Comment:
```suggestion
  const [readableValue, setReadableValue] = useState(false);
```
흐음 위의 변수명이 더 적합해 보이는데 변경 어떠신가요..?

Reply:
요거 근데 타입이 boolean이라 현재가 낫지 않나요?? 🤔
`readableValue`로 하면 읽혀야 하는 값을 그대로 넘겨줘야할 느낌?!